### DISCUSSION TITLE
520

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#917] [Comments]
# 520

MySQL 5.5에서 5.6으로 업데이트가 되면서 서브쿼리(Subquery) 성능 개선이 많이 이루어졌습니다. 이번 시간에는 MySQL 2개의 버전 (5.5, 5.6) 에서 서브쿼리를 통한 조회 (Select)와 Join에서의 조회간의 성능 차이를 비교해보겠습니다. MySQL의 정석과도 같은 Real MySQL 책이 MySQL 5.5 버전을 기준으로 하다보니 5.6 변경분에 대해서 별도로 포스팅하게 되었습니다. 0. 테스트 환경 테스트용 테이블은 2개를 만들었습니다. 메인 테이블 100만건 서브 테이블1 (인덱스 O) 1000건 서브 테이블2 (인덱스 X) 1000건 DDL 쿼리는 다음과 같습니다. 메인 테이블 -- 업데이트 대상 테이블 create table main_table ( id int not ..

https://jojoldu.tistory.com/520

**Comment by Jaeyoung22** [comment]
안녕하세요 동욱님! 우테코 5기 '리오'로 활동하고 있는 오영택이라고 합니다!
글도 항상 잘 읽고 있고 책에 해주신 싸인도 소중히 간직하고 있습니다! ㅎㅎ

다름이 아니라 사소하지만 해당 글에 수정할 부분이 있는 듯 해서 커멘트 남깁니다!
서브 테이블 중에 인덱스가 걸려 있지 않은 쪽에 인덱스가 있다고 설명이 되어있고,
인덱스가 걸린 쪽에 인덱스가 없다고 설명이 되어있어서 읽으면서 살짝 혼동이 있었습니다..!
